### PR TITLE
Add more options for issue type in "data-problem"

### PR DESCRIPTION
**Pull request by queengooborg**
This PR offers two additional dropdown options for a "data problem" issue's type, namely "missing compatibility data" and "missing specification".  These two are common occurrences.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/data-problem.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ body:
       options:
         - Incorrect support data (ex. Chrome says "86" but support was added in "40")
         - Browser bug (a bug with a feature that may impact site compatibility)
+        - Missing compatibility data
+        - Missing specification
```
Comment (janbrasna) [inline .github/ISSUE_TEMPLATE/data-problem.yml:30]: I'm trying to understand what cases fall under "Missing specification" exactly; maybe a more verbose description would remove this uncertainty?

- The entry doesn't have a `spec_url` but should?
- The entry doesn't have a `mdn_url` but should?
- The entry has `mdn_url` linking to a 404 not-yet-authored page which shloud be removed?
- The entry has `mdn_url` linking to a 404 not-yet-authored page that shloud be written?
- Something completely different and I'm inferring it just wrong…

**Review by janbrasna — COMMENTED** [review]
@queengooborg 👋 Trying to see it from a new contributor's perspective, I might need more explanation for the last type to apply it correctly:

File: .github/ISSUE_TEMPLATE/data-problem.yml (hunk 1 above)
Comment (queengooborg) [reply .github/ISSUE_TEMPLATE/data-problem.yml]: Thank you for your feedback!  I've just pushed an update to clarify better that it is referring to the `spec_url` property!

Comment (janbrasna) [reply .github/ISSUE_TEMPLATE/data-problem.yml]: That is the easiest of all the options (:) no varying opinions here;) 💣 thanks for clarification! Much appreciated, this should help lowering decision paralysis for newcomers.

**Pull request merged**